graphics/include/gz/common/MeshManager.hh

```diff
@@ -247,7 +247,7 @@ namespace gz
       /// \brief Sets the forceAssimp flag by reading the GZ_MESH_FORCE_ASSIMP
       /// environment variable. If forceAssimp true, MeshManager uses Assimp
       /// for loading all mesh formats, otherwise only for GLTF and FBX.
-      public: void SetAssimpEnvs();
+      public: void SetAssimpEnv();
```

Comment:
I recommend not changing the signature of this method, otherwise we need to tick-tock the method name to deprecate this one and create the new one.

Reply:
Okay, updating them.

Reply:
Done.